weave/trace/op.py

```diff
@@ -331,6 +333,15 @@ def _placeholder_call() -> Call:
     )
 
 
+def is_placeholder_call(call: Call) -> bool:
```

Comment:
if you are going to formalize this, I think it would be better to use a subclass as opposed to multiple value checks